src/writer.js

```diff
@@ -23,53 +24,71 @@ class Writer {
     if (trace.started.length === trace.finished.length) {
       const formattedTrace = trace.finished.map(format)
 
-      log.debug(() => `Encoding trace: ${JSON.stringify(formattedTrace)}`)
```

Comment:
We stop logging the encoded trace. We could still do this below in `flush()` if we think it's valuable.